Entity/AbstractCustomEntity.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Pim\Bundle\CustomEntityBundle\Entity;
 
+use DateTime;
```

Comment:
I don't see the point to have a `use DateTime;` only to avoid the backlash when using it.

Reply:
:+1: